js/background.js

```diff
@@ -69,10 +69,70 @@
         return accountManager;
     };
 
+    function rollDice(text) {
+        var percentage = parseInt(text, 10);
+
+        if (isNaN(percentage)) {
+            return false;
+        }
+
+        var roll = _.random(1, 100);
+        if (roll <= percentage) {
+            return true;
+        }
+
+        return false;
+    }
+
+    var UPGRADE_VALUE = 'feb-2018-upgrade-dice';
+    var UPGRADE_FLAG = window.UPGRADE_FLAG = 'feb-2018-upgrade-alert';
```

Comment:
Would it make sense to namespace `window.UPGRADE_FLAG` slightly to avoid clobbering `window` directly? Although I know we are phasing this all out…